doc/org/build/networking.md

```diff
@@ -0,0 +1,256 @@
+- [Intro](#orgca5b855)
+- [The `Networked` Component](#orgabca6d0)
+- [Message Types](#orge720d3c)
+  - [`CreateMessage`](#org4f63df9)
+  - [`UpdateMessage`](#orge698598)
+  - [`DeleteMessage`](#org8f17ac8)
+  - [`ClientJoin`](#orgf98374b)
+  - [`ClientLeave`](#orgebe8740)
+  - [`CreatorChange`](#org9b3f8a5)
+- [Eventual Consistency](#orga8173d0)
+- [Creating Networked Entities](#org62cc95d)
+- [Writing Networked Components](#orgd90eddd)
+- [Persisting Networked Entity State](#orgd0901c8)
+- [Networked Entity Hierarchies](#orge3d34eb)
+
+
+<a id="orgca5b855"></a>
+
+# Intro
+
+This document describes the way that entity state is networked across clients and optionally persisted to the database.
+
+Entity state includes things like what objects should be created by each client in a room, where those objects are, and various properties about them. `WebRTC` (voice, video, screenshare) connections are out of scope for this document: Those are handled separately by `dialog` and the `DialogAdapter`.
+
+Each client is connected to a reticulum node via a phoenix channel. The relevant channel for entity state networking is the `HubChannel`. Each client sends and receives `"nn"` messages containing entity state information over the `HubChannel`.
+
+Clients send messages at fixed intervals in the `networkSendSystem`. Messages are sent at fixed intervals (rather than anytime entity state is updated) so that frequently updated components (like `NetworkedTransform`) do not cause a client to flood the network with an unnecessary amount of `UpdateMessage` s.
+
+Clients receive messages outside of frame boundaries, and simply queue them for processing. Clients process queued messages each frame in the `networkReceiveSystem`.
+
+
+<a id="orgabca6d0"></a>
+
+# The `Networked` Component
```

Comment:
I think there seems to be a gap between in this section and the previous section. Readers may think "What is this component for?", "Is an entity with this component automatically synced with remote clients?", or anything else.

It would be better to let this section more descriptive, for example what it is for or why we start with explaining this component.